bindings/java/src/main/com/apple/foundationdb/Database.java

```diff
@@ -162,11 +162,11 @@ default Transaction createTransaction() {
 	double getMainThreadBusyness();
 
 	/**
-	 * Runs {@link #purgeBlobGranules(Function)} on the default executor.
+	 * Runs {@link #purgeBlobGranules(byte[] beginKey, byte[] endKey, boolean force)} on the default executor.
```

Comment:
Nit: I'm not really sure this makes sense, it's saying it's running the function signature on the default executor? I left it opaque because it's really just a passthrough function onto the default executor.

Reply:
I'm not smart enough to know the actual right answer here, but adding the actual method signature here allowed the build to work (and made my IDE happy ¯\_(ツ)_/¯ )

I think that javadoc (at least with JDK 11) does not understand the keyword `Function` in this context.